docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :feature: `1160` The Accounts & Balances page layout has been updated to increase usability. It is now split across three sub-pages: Blockchain Balances, Exchange Balances, Manual Balances (includes Fiat Balances). Exchange Balances is a new page where you will be able to see all of your asset balances for each connected exchange (previously this was only accessible from the Dashboard by clicking on an exchange). 
 * :bug: `1155` Fix the cryptocompate price queries of LUNA Terra
```

Comment:
Also add a changelog entry for the bug you fixed: https://github.com/rotki/rotki/issues/1140

Reply:
Added in next commit.